server/app/controllers/TaskSolver.scala

```diff
@@ -67,5 +100,9 @@ object TaskSolver {
 
   def sbt(command: String): Try[Boolean] = Try(Seq("sbt", command).! == 0)
 
-  lazy val sbtInstalled = sbt("sbtVersion").isSuccess // no exception, so sbt is in the PATH
+  // no exception, so sbt is in the PATH
+  lazy val sbtInstalled = sbt("--version").isSuccess
```

Comment:
We check that sbt installed by looking at sbtVersion or by looking at version of our application? --version return version of application:
[devgym] $ --version
[info] client/*:version
[info]  0.1-SNAPSHOT
[info] server/*:version
[info]  1.0-SNAPSHOT